hikari/impl/shard.py

```diff
@@ -279,6 +279,9 @@ async def connect(
                     proxy=proxy_settings.url,
                     proxy_headers=proxy_settings.headers,
                     url=url,
+                    # We manage these ourselves
+                    autoping=False,
```

Comment:
This is wrong, none of this logic handles PING or PONG events and without autoping this logic could break if discord started sending PINGs

Reply:
Discord's way of ping poing is by using heartbeats, but I do see and understand the concern in the case where they start sending ping pongs. Will close this then

Reply:
You can still disable autoclose though, that will improve semantics since the current logic does handle close codes.

Also ping/pong is a part of the spec and should be handled regardless of discord's current behaviour